src/main/java/uk/gov/companieshouse/api/strikeoffobjections/processor/ObjectionProcessor.java

```diff
@@ -106,6 +108,18 @@ private void validateObjectionStatus(Objection objection, String httpRequestId)
         }
     }
 
+    private void validateObjectionData(Objection objection, String httpRequestId) throws ServiceException {
+        if (objection.isDataEnteredByUserIncomplete()) {
```

Comment:
This addresses the main issue reported in OBJ-233 - fast-forwarding to the check your answers screen by changing the URL and submitting an Objection with incomplete data.